mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.erb

```diff
@@ -1053,6 +1093,42 @@ func expandSecurityPolicyMatchExpr(expr []interface{}) *compute.Expr {
 	}
 }
 
+func expandSecurityPolicyMatchExprOptions(exprOptions []interface{}) *compute.SecurityPolicyRuleMatcherExprOptions {
+	if len(exprOptions) == 0 || exprOptions[0] == nil {
+		return nil
+	}
+
+	data := exprOptions[0].(map[string]interface{})
+	return &compute.SecurityPolicyRuleMatcherExprOptions{
+		RecaptchaOptions: expandSecurityPolicyMatchExprOptionsRecaptchaOptions(data["recaptcha_options"].([]interface{})),
+	}
+}
+
+func expandSecurityPolicyMatchExprOptionsRecaptchaOptions(RecaptchaOptions []interface{}) *compute.SecurityPolicyRuleMatcherExprOptionsRecaptchaOptions {
+	if len(RecaptchaOptions) == 0 || RecaptchaOptions[0] == nil {
+		return nil
+	}
+
+	data := RecaptchaOptions[0].(map[string]interface{})
+
+  actionTokenKeysInterface := data["action_token_site_keys"].([]interface{})
+	actionTokenKeysString := make([]string, len(actionTokenKeysInterface))
+	for i, v := range actionTokenKeysInterface {
+		actionTokenKeysString[i] = v.(string)
+	}
+
+	sessionTokenKeysInterface := data["session_token_site_keys"].([]interface{})
+	sessionTokenKeysString := make([]string, len(sessionTokenKeysInterface))
```

Comment:
sessionTokenKeysString is a little confusing as it's an array. Can you make this just sessionTokenKeys?

Reply:
Same for actionTokenKeys